CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## ## [3.0.0] - 2023-02-07
+### Added
+- JSONAPI data member validation
```

Comment:
- Only "links" and "meta" are allowed besides id, type and attributes in "data". If anything else is present, an error is now raised.

makes it clearer what this is about and what can now potentially go wrong